### ISSUE TITLE
Confusing interface, UX problem?

### ISSUE THREAD

**Issue by ghost** [mazurio/bodyweight-fitness-android#44]
First, thanks for awesome app. Because it is open source I have purchased the PRO version.

I feel really dumb saying this, but I was really confused by the workout interface. There are buttons with same icons and the introduction tutorial slides do not show you exactly how it works. I wasn't sure how exactly do I log the workout.

![screenshot_20161027-141119](https://cloud.githubusercontent.com/assets/10860015/19767345/a81e669c-9c52-11e6-81f9-c0be8825340e.png)

I assume it works like this (correct me if I am wrong):
At the very top you have recommended reps/sets and with top buttons (- and +) you modify the current reps in a set and then you log the current set with an edit (pencil) button. If you made a mistake you can edit the saved reps/set with the bottom edit button (pencil). The plus button at the bottom is for more options. And that really confused the shit out of me at first.

This took me a while to understand, since at first I didn't know how exactly the above buttons log my workout. So simply I just used the bottom edit button and input the reps/sets manually just to be sure.

I'm not expert in UX field so these are only my suggestions/opinions:
- instead of a plus icon for more options use [three dots icon](https://material.io/icons/#ic_more_vert)
- for the "apply current set" button (at the top) use the [check icon](https://material.io/icons/#ic_check) instead of edit icon
- I think that the FAB is meant for primary action of that activity, so the two FABs that you use are maybe misused? This is not a hard rule, since Google often doesn't follow their guidelines.

**Comment by mazurio** [comment]
Hi @skmlcd - you are right with everything you have written down.

At the very top we have recommended reps/sets from section the exercise is in. There are two types of exercises (weighted / timed). Weighted exercises have a reps logger (shown above) where timed exercises have a timer. Everything is logged automatically as you go. You just press the button once but you can edit the values later.

I really like your suggestions when it comes to using three dots icon and check icon. I will look into using it when I am back from short holidays. I think the "+" button needs some improvements as there is rest timer coming soon.

Thanks,
Damian

**Comment by krokofant** [comment]
The edit button(but really the check button 😕 ) is really confusing. I always assume it's edit. Would be lovely is it was a check or if we want to be extra fancy a check with a little number top left inside the button. 

Anyway, it's been two months, are we set on how the buttons should look? If you don't have time would you accept a PR?

**Comment by mazurio** [comment]
Hey @krokofant - I have just finished adding SS. I am planning to look at UI after 6th of January but if you want/have time - feel free to open a PR. I will then have a look at how it looks. I was thinking about changing some buttons anyway.

**Comment by annawoodard** [comment]
> First, thanks for awesome app.

Seconded! This app is awesome, many thanks to @mazurio for all of the hard work!

> The edit button(but really the check button 😕 ) is really confusing. I always assume it's edit. Would be lovely is it was a check or if we want to be extra fancy a check with a little number top left inside the button.

:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

As a first time user, I agree with @krokofant and @skmlcd that the edit button is confusing.

**Comment by jonsaq** [comment]
Just downloaded and used for the first time, great functionality, thanks for putting this together @mazurio.

I agree that the UI/UX is a bit confusing and could be optimized with a few small tweaks, came here to say that.
I took a quick stab at a revised mockup to fix some of the UI issues, mainly information hierarchy and iconography. By no means a perfect fix, but something to think about. I think there could be some rethinking of the user flows (ex: starting a workout; It would be nice to see the workout overview (whatever the the top right icon's page is) before diving into the routine)

![mobile](https://cloud.githubusercontent.com/assets/4789198/21665930/6dd41270-d2a3-11e6-8046-3487ef35cfca.png)

Again, this is still really cool, and I was actually doing an Arch while whipping this up on my laptop. You rock @mazurio!

**Comment by hokster7** [comment]
Agreed UX needs work, but for a free open source project the current version is pretty fantastic. Good job @mazurio!
